package.json

```diff
@@ -17,7 +17,7 @@
   },
   "devDependencies": {
     "mocha": "2.2.4",
-    "nock": "0.48.1",
+    "nock": "^2.17.0",
```

Comment:
Can you please remove the `^`. I want to maintain reproducible/deterministic builds.